src/main/services/bs-launcher/steam-launcher.service.ts

```diff
@@ -77,8 +77,11 @@ export class SteamLauncherService extends AbstractLauncherService implements Sto
                 throw CustomError.fromError(new Error(`Path not exist : ${exePath}`), BSLaunchError.BS_NOT_FOUND);
             }
 
+            const staticConfigService = StaticConfigurationService.getInstance();
+            const launchVREnabled = await staticConfigService.get("disable-launch-vr");
```

Comment:
From a readability standpoint, this is very confusing. If "disable-launch-vr" is true then this will check if steam is running. Might want to use "enable-launch-vr" instead, or "enable-steamvr-launch" that other dev may get context that this is tied to SteamVR and Steam.

Reply:
I did that originally but changed it to match the formatting of hardware acceleration 💀

Alright that's easy enough to redo